src/minifluxapi.cpp

```diff
@@ -235,6 +235,22 @@ rsspp::Feed MinifluxApi::fetch_feed(const std::string& id, CurlHandle& cached_ha
 				item.content_encoded = entry["content"];
 			}
 
+			if (!entry["enclosures"].is_null() && entry["enclosures"].is_array()) {
+				for (const auto& enclosure : entry["enclosures"]) {
+					rsspp::Enclosure enc;
+
+					if (!enclosure["url"].is_null()) {
+						enc.url = enclosure["url"];
+					}
+
+					if (!enclosure["mime_type"].is_null()) {
+						enc.type = enclosure["mime_type"];
+					}
```

Comment:
Enclosures without URL make little sense, so I'd combine those `if`s into one.

Reply:
Just changed it. It will now only add the enclosure if it has a URL and MIME Type.

Not sure if the MIME Type requirement is necessary though.

Reply:
I think it's fine to have them in a single `if`: Newsboat only displays a single enclosure, and it uses the MIME type to decide which one to pick. That limitation should get removed eventually (#2638), at which point MIME type will become optional.